streams-contrib/streams-persist-graph/src/main/java/org/apache/streams/graph/GraphHttpPersistWriter.java

```diff
@@ -218,7 +217,7 @@ protected ObjectNode executePost(HttpPost httpPost) {
     @Override
     public void prepare(Object configurationObject) {
```

Comment:
prepare(Object) is defined by the interface and thus has to be implemented, but exact implementation details vary.  Most modules source component configurations at component creation (constructor) time as the runtimes are for the most part not set up to supply useful configurationObjects.